riscof/framework/test.py

```diff
@@ -298,19 +298,15 @@ def prod_isa(dut_isa, test_isa):
     isa = set([])
     last_prefix = ''
     atleast_1 = False
-
+    match = re.findall("(?P<prefix>RV(64|128|32)(I|E))",dut_isa)
```

Comment:
Do we need to add other Extensions here like (?P<prefix>RV(64|128|32)(I|E|F|D))?

Reply:
No. Its handled in the matching logic. This regex only handles the prefix of the isa i.e the base ISA. The base ISA of the dut is used in matching when the same test can be used for different base ISAs.